[27/Apr/2017 20:10:48] ERROR [django.request:256] Internal Server Error: /DynamfitRun
Traceback (most recent call last):
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/NANOMINE/Develop/mdcs-v1.5_rc1/dynamfit/views.py", line 220, in runmodel
    }, context_instance=RequestContext(request))
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/shortcuts.py", line 45, in render_to_response
    using=using)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader.py", line 115, in render_to_string
    template_name, context, context_instance, dirs, dictionary)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/engine.py", line 208, in render_to_string
    t = self.get_template(template_name, dirs)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/engine.py", line 167, in get_template
    template, origin = self.find_template(template_name, dirs)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/engine.py", line 141, in find_template
    source, display_name = loader(name, dirs)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loaders/base.py", line 13, in __call__
    return self.load_template(template_name, template_dirs)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loaders/base.py", line 23, in load_template
    template = Template(source, origin, template_name, self.engine)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 191, in __init__
    self.nodelist = engine.compile_string(template_string, origin)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/engine.py", line 261, in compile_string
    return parser.parse()
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 342, in parse
    compiled_result = compile_func(self, token)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader_tags.py", line 210, in do_extends
    nodelist = parser.parse()
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 342, in parse
    compiled_result = compile_func(self, token)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader_tags.py", line 184, in do_block
    nodelist = parser.parse(('endblock',))
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 318, in parse
    filter_expression = self.compile_filter(token.contents)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 424, in compile_filter
    return FilterExpression(token, self)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 640, in __init__
    "from '%s'" % (token[upto:], token))
TemplateSyntaxError: Could not parse the remainder: ' not job_id_num == 'not found'' from 'if not job_id_num == 'not found''
[27/Apr/2017 20:12:26] ERROR [django.request:256] Internal Server Error: /DynamfitRun
Traceback (most recent call last):
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/NANOMINE/Develop/mdcs-v1.5_rc1/dynamfit/views.py", line 221, in runmodel
    }, context_instance=RequestContext(request))
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/shortcuts.py", line 45, in render_to_response
    using=using)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader.py", line 115, in render_to_string
    template_name, context, context_instance, dirs, dictionary)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/engine.py", line 208, in render_to_string
    t = self.get_template(template_name, dirs)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/engine.py", line 167, in get_template
    template, origin = self.find_template(template_name, dirs)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/engine.py", line 141, in find_template
    source, display_name = loader(name, dirs)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loaders/base.py", line 13, in __call__
    return self.load_template(template_name, template_dirs)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loaders/base.py", line 23, in load_template
    template = Template(source, origin, template_name, self.engine)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 191, in __init__
    self.nodelist = engine.compile_string(template_string, origin)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/engine.py", line 261, in compile_string
    return parser.parse()
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 342, in parse
    compiled_result = compile_func(self, token)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader_tags.py", line 210, in do_extends
    nodelist = parser.parse()
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 342, in parse
    compiled_result = compile_func(self, token)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader_tags.py", line 184, in do_block
    nodelist = parser.parse(('endblock',))
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 318, in parse
    filter_expression = self.compile_filter(token.contents)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 424, in compile_filter
    return FilterExpression(token, self)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 640, in __init__
    "from '%s'" % (token[upto:], token))
TemplateSyntaxError: Could not parse the remainder: ' job_found' from 'if job_found'
[27/Apr/2017 20:13:01] ERROR [django.request:256] Internal Server Error: /DynamfitRun
Traceback (most recent call last):
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/NANOMINE/Develop/mdcs-v1.5_rc1/dynamfit/views.py", line 221, in runmodel
    }, context_instance=RequestContext(request))
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/shortcuts.py", line 45, in render_to_response
    using=using)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader.py", line 115, in render_to_string
    template_name, context, context_instance, dirs, dictionary)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/engine.py", line 208, in render_to_string
    t = self.get_template(template_name, dirs)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/engine.py", line 167, in get_template
    template, origin = self.find_template(template_name, dirs)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/engine.py", line 141, in find_template
    source, display_name = loader(name, dirs)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loaders/base.py", line 13, in __call__
    return self.load_template(template_name, template_dirs)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loaders/base.py", line 23, in load_template
    template = Template(source, origin, template_name, self.engine)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 191, in __init__
    self.nodelist = engine.compile_string(template_string, origin)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/engine.py", line 261, in compile_string
    return parser.parse()
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 342, in parse
    compiled_result = compile_func(self, token)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader_tags.py", line 210, in do_extends
    nodelist = parser.parse()
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 342, in parse
    compiled_result = compile_func(self, token)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader_tags.py", line 184, in do_block
    nodelist = parser.parse(('endblock',))
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 318, in parse
    filter_expression = self.compile_filter(token.contents)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 424, in compile_filter
    return FilterExpression(token, self)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 640, in __init__
    "from '%s'" % (token[upto:], token))
TemplateSyntaxError: Could not parse the remainder: '% if job_found %' from '% if job_found %'
[27/Apr/2017 20:16:06] WARNING [django.request:170] Not Found: /Dynamfit
[27/Apr/2017 21:41:15] ERROR [django.request:256] Internal Server Error: /descchar_view
Traceback (most recent call last):
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/NANOMINE/Develop/mdcs-v1.5_rc1/descchar/views.py", line 132, in check
    if int( tp.split(',')[-1] ) == int(count):
ValueError: invalid literal for int() with base 10: ''
[27/Apr/2017 21:41:51] ERROR [django.request:256] Internal Server Error: /DynamfitRun
Traceback (most recent call last):
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/NANOMINE/Develop/mdcs-v1.5_rc1/dynamfit/views.py", line 193, in runmodel
    if int( tp.split(',')[-2] ) == int(count):
ValueError: invalid literal for int() with base 10: ''
[27/Apr/2017 21:42:42] ERROR [django.request:256] Internal Server Error: /DynamfitRun
Traceback (most recent call last):
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/NANOMINE/Develop/mdcs-v1.5_rc1/dynamfit/views.py", line 193, in runmodel
    if int( tp.split(',')[-2] ) == int(count):
ValueError: invalid literal for int() with base 10: ''
[27/Apr/2017 21:43:18] ERROR [django.request:256] Internal Server Error: /DynamfitRun
Traceback (most recent call last):
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/NANOMINE/Develop/mdcs-v1.5_rc1/dynamfit/views.py", line 193, in runmodel
    if int( tp.split(',')[-2] ) == int(count):
ValueError: invalid literal for int() with base 10: ''
[27/Apr/2017 21:46:06] ERROR [django.request:256] Internal Server Error: /DynamfitRun
Traceback (most recent call last):
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/NANOMINE/Develop/mdcs-v1.5_rc1/dynamfit/views.py", line 193, in runmodel
    count = 0
ValueError: invalid literal for int() with base 10: ''
[27/Apr/2017 22:47:52] ERROR [django.request:256] Internal Server Error: /niblack
Traceback (most recent call last):
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/core/handlers/base.py", line 108, in get_response
    response = middleware_method(request)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/middleware/common.py", line 75, in process_request
    urlresolvers.is_valid_path("%s/" % request.path_info, urlconf)):
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/core/urlresolvers.py", line 646, in is_valid_path
    resolve(path, urlconf)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/core/urlresolvers.py", line 521, in resolve
    return get_resolver(urlconf).resolve(path)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/core/urlresolvers.py", line 367, in resolve
    sub_match = pattern.resolve(new_path)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/core/urlresolvers.py", line 239, in resolve
    return ResolverMatch(self.callback, args, kwargs, self.name)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/core/urlresolvers.py", line 246, in callback
    self._callback = get_callable(self._callback_str)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/utils/lru_cache.py", line 101, in wrapper
    result = user_function(*args, **kwds)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/core/urlresolvers.py", line 105, in get_callable
    mod = import_module(mod_name)
  File "/home/NANOMINE/anaconda2/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
  File "/home/NANOMINE/Develop/mdcs-v1.5_rc1/niblack/views.py", line 39
    form = DocumentForm() # A empty, unbound form
       ^
IndentationError: expected an indented block
[27/Apr/2017 23:03:35] ERROR [django.request:256] Internal Server Error: /niblack/
Traceback (most recent call last):
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/NANOMINE/Develop/mdcs-v1.5_rc1/niblack/views.py", line 66, in home
    context_instance=RequestContext(request))
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/shortcuts.py", line 45, in render_to_response
    using=using)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader.py", line 115, in render_to_string
    template_name, context, context_instance, dirs, dictionary)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/engine.py", line 221, in render_to_string
    return t.render(context_instance)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 210, in render
    return self._render(context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 202, in _render
    return self.nodelist.render(context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 905, in render
    bit = self.render_node(node, context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 919, in render_node
    return node.render(context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader_tags.py", line 135, in render
    return compiled_parent._render(context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 202, in _render
    return self.nodelist.render(context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 905, in render
    bit = self.render_node(node, context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 919, in render_node
    return node.render(context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader_tags.py", line 135, in render
    return compiled_parent._render(context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 202, in _render
    return self.nodelist.render(context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 905, in render
    bit = self.render_node(node, context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 919, in render_node
    return node.render(context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader_tags.py", line 135, in render
    return compiled_parent._render(context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 202, in _render
    return self.nodelist.render(context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 905, in render
    bit = self.render_node(node, context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 919, in render_node
    return node.render(context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader_tags.py", line 65, in render
    result = block.nodelist.render(context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 905, in render
    bit = self.render_node(node, context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 919, in render_node
    return node.render(context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader_tags.py", line 65, in render
    result = block.nodelist.render(context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 905, in render
    bit = self.render_node(node, context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 919, in render_node
    return node.render(context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader_tags.py", line 65, in render
    result = block.nodelist.render(context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 905, in render
    bit = self.render_node(node, context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 919, in render_node
    return node.render(context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/defaulttags.py", line 328, in render
    if match:
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/db/models/query.py", line 170, in __nonzero__
    return type(self).__bool__(self)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/db/models/query.py", line 166, in __bool__
    self._fetch_all()
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/db/models/query.py", line 965, in _fetch_all
    self._result_cache = list(self.iterator())
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/db/models/query.py", line 238, in iterator
    results = compiler.execute_sql()
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/db/models/sql/compiler.py", line 840, in execute_sql
    cursor.execute(sql, params)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/db/backends/utils.py", line 79, in execute
    return super(CursorDebugWrapper, self).execute(sql, params)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/db/backends/utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/db/utils.py", line 98, in __exit__
    six.reraise(dj_exc_type, dj_exc_value, traceback)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/db/backends/utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/db/backends/sqlite3/base.py", line 318, in execute
    return Database.Cursor.execute(self, query, params)
OperationalError: no such table: niblack_document
[28/Apr/2017 00:31:49] DEBUG [paramiko.transport:282] starting thread (client mode): 0x18015190L
[28/Apr/2017 00:31:49] DEBUG [paramiko.transport:282] Local version/idstring: SSH-2.0-paramiko_2.1.2
[28/Apr/2017 00:31:49] DEBUG [paramiko.transport:282] Remote version/idstring: SSH-2.0-OpenSSH_6.6.1
[28/Apr/2017 00:31:49] INFO [paramiko.transport:282] Connected (version 2.0, client OpenSSH_6.6.1)
[28/Apr/2017 00:31:49] DEBUG [paramiko.transport:282] kex algos:[u'curve25519-sha256@libssh.org', u'ecdh-sha2-nistp256', u'ecdh-sha2-nistp384', u'ecdh-sha2-nistp521', u'diffie-hellman-group-exchange-sha256', u'diffie-hellman-group-exchange-sha1', u'diffie-hellman-group14-sha1', u'diffie-hellman-group1-sha1'] server key:[u'ssh-rsa', u'ecdsa-sha2-nistp256', u'ssh-ed25519'] client encrypt:[u'aes128-ctr', u'aes192-ctr', u'aes256-ctr', u'arcfour256', u'arcfour128', u'aes128-gcm@openssh.com', u'aes256-gcm@openssh.com', u'chacha20-poly1305@openssh.com', u'aes128-cbc', u'3des-cbc', u'blowfish-cbc', u'cast128-cbc', u'aes192-cbc', u'aes256-cbc', u'arcfour', u'rijndael-cbc@lysator.liu.se'] server encrypt:[u'aes128-ctr', u'aes192-ctr', u'aes256-ctr', u'arcfour256', u'arcfour128', u'aes128-gcm@openssh.com', u'aes256-gcm@openssh.com', u'chacha20-poly1305@openssh.com', u'aes128-cbc', u'3des-cbc', u'blowfish-cbc', u'cast128-cbc', u'aes192-cbc', u'aes256-cbc', u'arcfour', u'rijndael-cbc@lysator.liu.se'] client mac:[u'hmac-md5-etm@openssh.com', u'hmac-sha1-etm@openssh.com', u'umac-64-etm@openssh.com', u'umac-128-etm@openssh.com', u'hmac-sha2-256-etm@openssh.com', u'hmac-sha2-512-etm@openssh.com', u'hmac-ripemd160-etm@openssh.com', u'hmac-sha1-96-etm@openssh.com', u'hmac-md5-96-etm@openssh.com', u'hmac-md5', u'hmac-sha1', u'umac-64@openssh.com', u'umac-128@openssh.com', u'hmac-sha2-256', u'hmac-sha2-512', u'hmac-ripemd160', u'hmac-ripemd160@openssh.com', u'hmac-sha1-96', u'hmac-md5-96'] server mac:[u'hmac-md5-etm@openssh.com', u'hmac-sha1-etm@openssh.com', u'umac-64-etm@openssh.com', u'umac-128-etm@openssh.com', u'hmac-sha2-256-etm@openssh.com', u'hmac-sha2-512-etm@openssh.com', u'hmac-ripemd160-etm@openssh.com', u'hmac-sha1-96-etm@openssh.com', u'hmac-md5-96-etm@openssh.com', u'hmac-md5', u'hmac-sha1', u'umac-64@openssh.com', u'umac-128@openssh.com', u'hmac-sha2-256', u'hmac-sha2-512', u'hmac-ripemd160', u'hmac-ripemd160@openssh.com', u'hmac-sha1-96', u'hmac-md5-96'] client compress:[u'none', u'zlib@openssh.com'] server compress:[u'none', u'zlib@openssh.com'] client lang:[u''] server lang:[u''] kex follows?False
[28/Apr/2017 00:31:49] DEBUG [paramiko.transport:282] Kex agreed: diffie-hellman-group1-sha1
[28/Apr/2017 00:31:49] DEBUG [paramiko.transport:282] Cipher agreed: aes128-ctr
[28/Apr/2017 00:31:49] DEBUG [paramiko.transport:282] MAC agreed: hmac-sha2-256
[28/Apr/2017 00:31:49] DEBUG [paramiko.transport:282] Compression agreed: none
[28/Apr/2017 00:31:49] DEBUG [paramiko.transport:282] kex engine KexGroup1 specified hash_algo <built-in function openssl_sha1>
[28/Apr/2017 00:31:49] DEBUG [paramiko.transport:282] Switch to new keys ...
[28/Apr/2017 00:31:49] DEBUG [paramiko.transport:282] Adding ssh-rsa host key for 129.105.90.149: 3be932e37fd6c127bb6880a6ce0bab88
[28/Apr/2017 00:31:49] DEBUG [paramiko.transport:282] Trying discovered key 175016a8e15c3b3b3130d9f0396baec2 in /home/NANOMINE/.ssh/id_rsa
[28/Apr/2017 00:31:49] DEBUG [paramiko.transport:282] userauth is OK
[28/Apr/2017 00:31:49] INFO [paramiko.transport:282] Authentication (publickey) successful!
[28/Apr/2017 00:31:49] DEBUG [paramiko.transport:282] [chan 0] Max packet in: 32768 bytes
[28/Apr/2017 00:31:49] DEBUG [paramiko.transport:282] [chan 0] Max packet out: 32768 bytes
[28/Apr/2017 00:31:49] DEBUG [paramiko.transport:282] Secsh channel 0 opened.
[28/Apr/2017 00:31:50] DEBUG [paramiko.transport:282] [chan 0] Sesch channel 0 request ok
[28/Apr/2017 00:31:50] DEBUG [paramiko.transport:282] [chan 1] Max packet in: 32768 bytes
[28/Apr/2017 00:31:50] DEBUG [paramiko.transport:282] [chan 0] EOF received (0)
[28/Apr/2017 00:31:50] DEBUG [paramiko.transport:282] [chan 0] EOF sent (0)
[28/Apr/2017 00:31:50] DEBUG [paramiko.transport:282] [chan 1] Max packet out: 32768 bytes
[28/Apr/2017 00:31:50] DEBUG [paramiko.transport:282] Secsh channel 1 opened.
[28/Apr/2017 00:31:50] DEBUG [paramiko.transport:282] [chan 1] Sesch channel 1 request ok
[28/Apr/2017 00:31:50] DEBUG [paramiko.transport:282] [chan 2] Max packet in: 32768 bytes
[28/Apr/2017 00:31:50] DEBUG [paramiko.transport:282] [chan 1] EOF received (1)
[28/Apr/2017 00:31:50] DEBUG [paramiko.transport:282] [chan 1] EOF sent (1)
[28/Apr/2017 00:31:50] DEBUG [paramiko.transport:282] [chan 2] Max packet out: 32768 bytes
[28/Apr/2017 00:31:50] DEBUG [paramiko.transport:282] Secsh channel 2 opened.
[28/Apr/2017 00:31:50] DEBUG [paramiko.transport:282] [chan 2] Sesch channel 2 request ok
[28/Apr/2017 00:31:50] ERROR [django.request:256] Internal Server Error: /niblack/
Traceback (most recent call last):
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/NANOMINE/Develop/mdcs-v1.5_rc1/niblack/views.py", line 75, in home
    os.sytem('scp '+DocPath+'/*.TIF '+SCPSink+run_id+'/')
AttributeError: 'module' object has no attribute 'sytem'
[28/Apr/2017 00:31:50] DEBUG [paramiko.transport:282] [chan 2] EOF received (2)
[28/Apr/2017 00:31:50] DEBUG [paramiko.transport:282] [chan 2] EOF sent (2)
[28/Apr/2017 00:32:04] DEBUG [paramiko.transport:282] EOF in transport thread
[28/Apr/2017 00:32:18] DEBUG [paramiko.transport:282] starting thread (client mode): 0xffcdeed0L
[28/Apr/2017 00:32:18] DEBUG [paramiko.transport:282] Local version/idstring: SSH-2.0-paramiko_2.1.2
[28/Apr/2017 00:32:18] DEBUG [paramiko.transport:282] Remote version/idstring: SSH-2.0-OpenSSH_6.6.1
[28/Apr/2017 00:32:18] INFO [paramiko.transport:282] Connected (version 2.0, client OpenSSH_6.6.1)
[28/Apr/2017 00:32:18] DEBUG [paramiko.transport:282] kex algos:[u'curve25519-sha256@libssh.org', u'ecdh-sha2-nistp256', u'ecdh-sha2-nistp384', u'ecdh-sha2-nistp521', u'diffie-hellman-group-exchange-sha256', u'diffie-hellman-group-exchange-sha1', u'diffie-hellman-group14-sha1', u'diffie-hellman-group1-sha1'] server key:[u'ssh-rsa', u'ecdsa-sha2-nistp256', u'ssh-ed25519'] client encrypt:[u'aes128-ctr', u'aes192-ctr', u'aes256-ctr', u'arcfour256', u'arcfour128', u'aes128-gcm@openssh.com', u'aes256-gcm@openssh.com', u'chacha20-poly1305@openssh.com', u'aes128-cbc', u'3des-cbc', u'blowfish-cbc', u'cast128-cbc', u'aes192-cbc', u'aes256-cbc', u'arcfour', u'rijndael-cbc@lysator.liu.se'] server encrypt:[u'aes128-ctr', u'aes192-ctr', u'aes256-ctr', u'arcfour256', u'arcfour128', u'aes128-gcm@openssh.com', u'aes256-gcm@openssh.com', u'chacha20-poly1305@openssh.com', u'aes128-cbc', u'3des-cbc', u'blowfish-cbc', u'cast128-cbc', u'aes192-cbc', u'aes256-cbc', u'arcfour', u'rijndael-cbc@lysator.liu.se'] client mac:[u'hmac-md5-etm@openssh.com', u'hmac-sha1-etm@openssh.com', u'umac-64-etm@openssh.com', u'umac-128-etm@openssh.com', u'hmac-sha2-256-etm@openssh.com', u'hmac-sha2-512-etm@openssh.com', u'hmac-ripemd160-etm@openssh.com', u'hmac-sha1-96-etm@openssh.com', u'hmac-md5-96-etm@openssh.com', u'hmac-md5', u'hmac-sha1', u'umac-64@openssh.com', u'umac-128@openssh.com', u'hmac-sha2-256', u'hmac-sha2-512', u'hmac-ripemd160', u'hmac-ripemd160@openssh.com', u'hmac-sha1-96', u'hmac-md5-96'] server mac:[u'hmac-md5-etm@openssh.com', u'hmac-sha1-etm@openssh.com', u'umac-64-etm@openssh.com', u'umac-128-etm@openssh.com', u'hmac-sha2-256-etm@openssh.com', u'hmac-sha2-512-etm@openssh.com', u'hmac-ripemd160-etm@openssh.com', u'hmac-sha1-96-etm@openssh.com', u'hmac-md5-96-etm@openssh.com', u'hmac-md5', u'hmac-sha1', u'umac-64@openssh.com', u'umac-128@openssh.com', u'hmac-sha2-256', u'hmac-sha2-512', u'hmac-ripemd160', u'hmac-ripemd160@openssh.com', u'hmac-sha1-96', u'hmac-md5-96'] client compress:[u'none', u'zlib@openssh.com'] server compress:[u'none', u'zlib@openssh.com'] client lang:[u''] server lang:[u''] kex follows?False
[28/Apr/2017 00:32:18] DEBUG [paramiko.transport:282] Kex agreed: diffie-hellman-group1-sha1
[28/Apr/2017 00:32:18] DEBUG [paramiko.transport:282] Cipher agreed: aes128-ctr
[28/Apr/2017 00:32:18] DEBUG [paramiko.transport:282] MAC agreed: hmac-sha2-256
[28/Apr/2017 00:32:18] DEBUG [paramiko.transport:282] Compression agreed: none
[28/Apr/2017 00:32:18] DEBUG [paramiko.transport:282] kex engine KexGroup1 specified hash_algo <built-in function openssl_sha1>
[28/Apr/2017 00:32:18] DEBUG [paramiko.transport:282] Switch to new keys ...
[28/Apr/2017 00:32:18] DEBUG [paramiko.transport:282] Adding ssh-rsa host key for 129.105.90.149: 3be932e37fd6c127bb6880a6ce0bab88
[28/Apr/2017 00:32:18] DEBUG [paramiko.transport:282] Trying discovered key 175016a8e15c3b3b3130d9f0396baec2 in /home/NANOMINE/.ssh/id_rsa
[28/Apr/2017 00:32:19] DEBUG [paramiko.transport:282] userauth is OK
[28/Apr/2017 00:32:19] INFO [paramiko.transport:282] Authentication (publickey) successful!
[28/Apr/2017 00:32:19] DEBUG [paramiko.transport:282] [chan 0] Max packet in: 32768 bytes
[28/Apr/2017 00:32:19] DEBUG [paramiko.transport:282] [chan 0] Max packet out: 32768 bytes
[28/Apr/2017 00:32:19] DEBUG [paramiko.transport:282] Secsh channel 0 opened.
[28/Apr/2017 00:32:19] DEBUG [paramiko.transport:282] [chan 0] Sesch channel 0 request ok
[28/Apr/2017 00:32:19] DEBUG [paramiko.transport:282] [chan 1] Max packet in: 32768 bytes
[28/Apr/2017 00:32:19] DEBUG [paramiko.transport:282] [chan 0] EOF received (0)
[28/Apr/2017 00:32:19] DEBUG [paramiko.transport:282] [chan 0] EOF sent (0)
[28/Apr/2017 00:32:19] DEBUG [paramiko.transport:282] [chan 1] Max packet out: 32768 bytes
[28/Apr/2017 00:32:19] DEBUG [paramiko.transport:282] Secsh channel 1 opened.
[28/Apr/2017 00:32:19] DEBUG [paramiko.transport:282] [chan 1] Sesch channel 1 request ok
[28/Apr/2017 00:32:19] DEBUG [paramiko.transport:282] [chan 2] Max packet in: 32768 bytes
[28/Apr/2017 00:32:19] DEBUG [paramiko.transport:282] [chan 1] EOF received (1)
[28/Apr/2017 00:32:19] DEBUG [paramiko.transport:282] [chan 1] EOF sent (1)
[28/Apr/2017 00:32:19] DEBUG [paramiko.transport:282] [chan 2] Max packet out: 32768 bytes
[28/Apr/2017 00:32:19] DEBUG [paramiko.transport:282] Secsh channel 2 opened.
[28/Apr/2017 00:32:19] DEBUG [paramiko.transport:282] [chan 2] Sesch channel 2 request ok
[28/Apr/2017 00:32:19] DEBUG [paramiko.transport:282] [chan 2] EOF received (2)
[28/Apr/2017 00:32:19] DEBUG [paramiko.transport:282] [chan 2] EOF sent (2)
[28/Apr/2017 00:32:20] DEBUG [paramiko.transport:282] [chan 3] Max packet in: 32768 bytes
[28/Apr/2017 00:32:20] DEBUG [paramiko.transport:282] [chan 3] Max packet out: 32768 bytes
[28/Apr/2017 00:32:20] DEBUG [paramiko.transport:282] Secsh channel 3 opened.
[28/Apr/2017 00:32:20] DEBUG [paramiko.transport:282] [chan 3] Sesch channel 3 request ok
[28/Apr/2017 00:32:20] DEBUG [paramiko.transport:282] [chan 4] Max packet in: 32768 bytes
[28/Apr/2017 00:32:20] DEBUG [paramiko.transport:282] [chan 4] Max packet out: 32768 bytes
[28/Apr/2017 00:32:20] DEBUG [paramiko.transport:282] Secsh channel 4 opened.
[28/Apr/2017 00:32:20] DEBUG [paramiko.transport:282] [chan 4] Sesch channel 4 request ok
[28/Apr/2017 00:32:20] DEBUG [paramiko.transport:282] [chan 3] EOF sent (3)
[28/Apr/2017 00:32:20] DEBUG [paramiko.transport:282] [chan 4] EOF received (4)
[28/Apr/2017 00:32:20] DEBUG [paramiko.transport:282] [chan 4] EOF sent (4)
[28/Apr/2017 00:32:22] DEBUG [paramiko.transport:282] Ignoring message for dead channel 3
[28/Apr/2017 00:32:22] DEBUG [paramiko.transport:282] Ignoring message for dead channel 3
[28/Apr/2017 00:33:29] DEBUG [paramiko.transport:282] EOF in transport thread
[28/Apr/2017 00:33:56] DEBUG [paramiko.transport:282] starting thread (client mode): 0x5b393910L
[28/Apr/2017 00:33:56] DEBUG [paramiko.transport:282] Local version/idstring: SSH-2.0-paramiko_2.1.2
[28/Apr/2017 00:33:56] DEBUG [paramiko.transport:282] Remote version/idstring: SSH-2.0-OpenSSH_6.6.1
[28/Apr/2017 00:33:56] INFO [paramiko.transport:282] Connected (version 2.0, client OpenSSH_6.6.1)
[28/Apr/2017 00:33:57] DEBUG [paramiko.transport:282] kex algos:[u'curve25519-sha256@libssh.org', u'ecdh-sha2-nistp256', u'ecdh-sha2-nistp384', u'ecdh-sha2-nistp521', u'diffie-hellman-group-exchange-sha256', u'diffie-hellman-group-exchange-sha1', u'diffie-hellman-group14-sha1', u'diffie-hellman-group1-sha1'] server key:[u'ssh-rsa', u'ecdsa-sha2-nistp256', u'ssh-ed25519'] client encrypt:[u'aes128-ctr', u'aes192-ctr', u'aes256-ctr', u'arcfour256', u'arcfour128', u'aes128-gcm@openssh.com', u'aes256-gcm@openssh.com', u'chacha20-poly1305@openssh.com', u'aes128-cbc', u'3des-cbc', u'blowfish-cbc', u'cast128-cbc', u'aes192-cbc', u'aes256-cbc', u'arcfour', u'rijndael-cbc@lysator.liu.se'] server encrypt:[u'aes128-ctr', u'aes192-ctr', u'aes256-ctr', u'arcfour256', u'arcfour128', u'aes128-gcm@openssh.com', u'aes256-gcm@openssh.com', u'chacha20-poly1305@openssh.com', u'aes128-cbc', u'3des-cbc', u'blowfish-cbc', u'cast128-cbc', u'aes192-cbc', u'aes256-cbc', u'arcfour', u'rijndael-cbc@lysator.liu.se'] client mac:[u'hmac-md5-etm@openssh.com', u'hmac-sha1-etm@openssh.com', u'umac-64-etm@openssh.com', u'umac-128-etm@openssh.com', u'hmac-sha2-256-etm@openssh.com', u'hmac-sha2-512-etm@openssh.com', u'hmac-ripemd160-etm@openssh.com', u'hmac-sha1-96-etm@openssh.com', u'hmac-md5-96-etm@openssh.com', u'hmac-md5', u'hmac-sha1', u'umac-64@openssh.com', u'umac-128@openssh.com', u'hmac-sha2-256', u'hmac-sha2-512', u'hmac-ripemd160', u'hmac-ripemd160@openssh.com', u'hmac-sha1-96', u'hmac-md5-96'] server mac:[u'hmac-md5-etm@openssh.com', u'hmac-sha1-etm@openssh.com', u'umac-64-etm@openssh.com', u'umac-128-etm@openssh.com', u'hmac-sha2-256-etm@openssh.com', u'hmac-sha2-512-etm@openssh.com', u'hmac-ripemd160-etm@openssh.com', u'hmac-sha1-96-etm@openssh.com', u'hmac-md5-96-etm@openssh.com', u'hmac-md5', u'hmac-sha1', u'umac-64@openssh.com', u'umac-128@openssh.com', u'hmac-sha2-256', u'hmac-sha2-512', u'hmac-ripemd160', u'hmac-ripemd160@openssh.com', u'hmac-sha1-96', u'hmac-md5-96'] client compress:[u'none', u'zlib@openssh.com'] server compress:[u'none', u'zlib@openssh.com'] client lang:[u''] server lang:[u''] kex follows?False
[28/Apr/2017 00:33:57] DEBUG [paramiko.transport:282] Kex agreed: diffie-hellman-group1-sha1
[28/Apr/2017 00:33:57] DEBUG [paramiko.transport:282] Cipher agreed: aes128-ctr
[28/Apr/2017 00:33:57] DEBUG [paramiko.transport:282] MAC agreed: hmac-sha2-256
[28/Apr/2017 00:33:57] DEBUG [paramiko.transport:282] Compression agreed: none
[28/Apr/2017 00:33:57] DEBUG [paramiko.transport:282] kex engine KexGroup1 specified hash_algo <built-in function openssl_sha1>
[28/Apr/2017 00:33:57] DEBUG [paramiko.transport:282] Switch to new keys ...
[28/Apr/2017 00:33:57] DEBUG [paramiko.transport:282] Adding ssh-rsa host key for 129.105.90.149: 3be932e37fd6c127bb6880a6ce0bab88
[28/Apr/2017 00:33:57] DEBUG [paramiko.transport:282] Trying discovered key 175016a8e15c3b3b3130d9f0396baec2 in /home/NANOMINE/.ssh/id_rsa
[28/Apr/2017 00:33:57] DEBUG [paramiko.transport:282] userauth is OK
[28/Apr/2017 00:33:57] INFO [paramiko.transport:282] Authentication (publickey) successful!
[28/Apr/2017 00:33:57] DEBUG [paramiko.transport:282] [chan 0] Max packet in: 32768 bytes
[28/Apr/2017 00:33:57] DEBUG [paramiko.transport:282] [chan 0] Max packet out: 32768 bytes
[28/Apr/2017 00:33:57] DEBUG [paramiko.transport:282] Secsh channel 0 opened.
[28/Apr/2017 00:33:57] DEBUG [paramiko.transport:282] [chan 0] Sesch channel 0 request ok
[28/Apr/2017 00:33:57] DEBUG [paramiko.transport:282] [chan 1] Max packet in: 32768 bytes
[28/Apr/2017 00:33:57] DEBUG [paramiko.transport:282] [chan 0] EOF received (0)
[28/Apr/2017 00:33:57] DEBUG [paramiko.transport:282] [chan 0] EOF sent (0)
[28/Apr/2017 00:33:57] DEBUG [paramiko.transport:282] [chan 1] Max packet out: 32768 bytes
[28/Apr/2017 00:33:57] DEBUG [paramiko.transport:282] Secsh channel 1 opened.
[28/Apr/2017 00:33:57] DEBUG [paramiko.transport:282] [chan 1] Sesch channel 1 request ok
[28/Apr/2017 00:33:57] DEBUG [paramiko.transport:282] [chan 2] Max packet in: 32768 bytes
[28/Apr/2017 00:33:57] DEBUG [paramiko.transport:282] [chan 1] EOF received (1)
[28/Apr/2017 00:33:57] DEBUG [paramiko.transport:282] [chan 1] EOF sent (1)
[28/Apr/2017 00:33:57] DEBUG [paramiko.transport:282] [chan 2] Max packet out: 32768 bytes
[28/Apr/2017 00:33:57] DEBUG [paramiko.transport:282] Secsh channel 2 opened.
[28/Apr/2017 00:33:57] DEBUG [paramiko.transport:282] [chan 2] Sesch channel 2 request ok
[28/Apr/2017 00:33:57] DEBUG [paramiko.transport:282] [chan 2] EOF received (2)
[28/Apr/2017 00:33:57] DEBUG [paramiko.transport:282] [chan 2] EOF sent (2)
[28/Apr/2017 00:33:58] DEBUG [paramiko.transport:282] [chan 3] Max packet in: 32768 bytes
[28/Apr/2017 00:33:58] DEBUG [paramiko.transport:282] [chan 3] Max packet out: 32768 bytes
[28/Apr/2017 00:33:58] DEBUG [paramiko.transport:282] Secsh channel 3 opened.
[28/Apr/2017 00:33:58] DEBUG [paramiko.transport:282] [chan 3] Sesch channel 3 request ok
[28/Apr/2017 00:33:58] DEBUG [paramiko.transport:282] [chan 4] Max packet in: 32768 bytes
[28/Apr/2017 00:33:59] DEBUG [paramiko.transport:282] [chan 4] Max packet out: 32768 bytes
[28/Apr/2017 00:33:59] DEBUG [paramiko.transport:282] Secsh channel 4 opened.
[28/Apr/2017 00:33:59] DEBUG [paramiko.transport:282] [chan 4] Sesch channel 4 request ok
[28/Apr/2017 00:33:59] DEBUG [paramiko.transport:282] [chan 3] EOF sent (3)
[28/Apr/2017 00:33:59] DEBUG [paramiko.transport:282] [chan 4] EOF received (4)
[28/Apr/2017 00:33:59] DEBUG [paramiko.transport:282] [chan 4] EOF sent (4)
[28/Apr/2017 00:34:01] DEBUG [paramiko.transport:282] Ignoring message for dead channel 3
[28/Apr/2017 00:34:01] DEBUG [paramiko.transport:282] Ignoring message for dead channel 3
[28/Apr/2017 00:37:42] DEBUG [paramiko.transport:282] starting thread (client mode): 0x5a894ad0L
[28/Apr/2017 00:37:42] DEBUG [paramiko.transport:282] Local version/idstring: SSH-2.0-paramiko_2.1.2
[28/Apr/2017 00:37:43] DEBUG [paramiko.transport:282] Remote version/idstring: SSH-2.0-OpenSSH_6.6.1
[28/Apr/2017 00:37:43] INFO [paramiko.transport:282] Connected (version 2.0, client OpenSSH_6.6.1)
[28/Apr/2017 00:37:43] DEBUG [paramiko.transport:282] kex algos:[u'curve25519-sha256@libssh.org', u'ecdh-sha2-nistp256', u'ecdh-sha2-nistp384', u'ecdh-sha2-nistp521', u'diffie-hellman-group-exchange-sha256', u'diffie-hellman-group-exchange-sha1', u'diffie-hellman-group14-sha1', u'diffie-hellman-group1-sha1'] server key:[u'ssh-rsa', u'ecdsa-sha2-nistp256', u'ssh-ed25519'] client encrypt:[u'aes128-ctr', u'aes192-ctr', u'aes256-ctr', u'arcfour256', u'arcfour128', u'aes128-gcm@openssh.com', u'aes256-gcm@openssh.com', u'chacha20-poly1305@openssh.com', u'aes128-cbc', u'3des-cbc', u'blowfish-cbc', u'cast128-cbc', u'aes192-cbc', u'aes256-cbc', u'arcfour', u'rijndael-cbc@lysator.liu.se'] server encrypt:[u'aes128-ctr', u'aes192-ctr', u'aes256-ctr', u'arcfour256', u'arcfour128', u'aes128-gcm@openssh.com', u'aes256-gcm@openssh.com', u'chacha20-poly1305@openssh.com', u'aes128-cbc', u'3des-cbc', u'blowfish-cbc', u'cast128-cbc', u'aes192-cbc', u'aes256-cbc', u'arcfour', u'rijndael-cbc@lysator.liu.se'] client mac:[u'hmac-md5-etm@openssh.com', u'hmac-sha1-etm@openssh.com', u'umac-64-etm@openssh.com', u'umac-128-etm@openssh.com', u'hmac-sha2-256-etm@openssh.com', u'hmac-sha2-512-etm@openssh.com', u'hmac-ripemd160-etm@openssh.com', u'hmac-sha1-96-etm@openssh.com', u'hmac-md5-96-etm@openssh.com', u'hmac-md5', u'hmac-sha1', u'umac-64@openssh.com', u'umac-128@openssh.com', u'hmac-sha2-256', u'hmac-sha2-512', u'hmac-ripemd160', u'hmac-ripemd160@openssh.com', u'hmac-sha1-96', u'hmac-md5-96'] server mac:[u'hmac-md5-etm@openssh.com', u'hmac-sha1-etm@openssh.com', u'umac-64-etm@openssh.com', u'umac-128-etm@openssh.com', u'hmac-sha2-256-etm@openssh.com', u'hmac-sha2-512-etm@openssh.com', u'hmac-ripemd160-etm@openssh.com', u'hmac-sha1-96-etm@openssh.com', u'hmac-md5-96-etm@openssh.com', u'hmac-md5', u'hmac-sha1', u'umac-64@openssh.com', u'umac-128@openssh.com', u'hmac-sha2-256', u'hmac-sha2-512', u'hmac-ripemd160', u'hmac-ripemd160@openssh.com', u'hmac-sha1-96', u'hmac-md5-96'] client compress:[u'none', u'zlib@openssh.com'] server compress:[u'none', u'zlib@openssh.com'] client lang:[u''] server lang:[u''] kex follows?False
[28/Apr/2017 00:37:43] DEBUG [paramiko.transport:282] Kex agreed: diffie-hellman-group1-sha1
[28/Apr/2017 00:37:43] DEBUG [paramiko.transport:282] Cipher agreed: aes128-ctr
[28/Apr/2017 00:37:43] DEBUG [paramiko.transport:282] MAC agreed: hmac-sha2-256
[28/Apr/2017 00:37:43] DEBUG [paramiko.transport:282] Compression agreed: none
[28/Apr/2017 00:37:43] DEBUG [paramiko.transport:282] kex engine KexGroup1 specified hash_algo <built-in function openssl_sha1>
[28/Apr/2017 00:37:43] DEBUG [paramiko.transport:282] Switch to new keys ...
[28/Apr/2017 00:37:43] DEBUG [paramiko.transport:282] Adding ssh-rsa host key for 129.105.90.149: 3be932e37fd6c127bb6880a6ce0bab88
[28/Apr/2017 00:37:43] DEBUG [paramiko.transport:282] Trying discovered key 175016a8e15c3b3b3130d9f0396baec2 in /home/NANOMINE/.ssh/id_rsa
[28/Apr/2017 00:37:43] DEBUG [paramiko.transport:282] userauth is OK
[28/Apr/2017 00:37:43] INFO [paramiko.transport:282] Authentication (publickey) successful!
[28/Apr/2017 00:37:43] DEBUG [paramiko.transport:282] [chan 0] Max packet in: 32768 bytes
[28/Apr/2017 00:37:43] DEBUG [paramiko.transport:282] [chan 0] Max packet out: 32768 bytes
[28/Apr/2017 00:37:43] DEBUG [paramiko.transport:282] Secsh channel 0 opened.
[28/Apr/2017 00:37:43] DEBUG [paramiko.transport:282] [chan 0] Sesch channel 0 request ok
[28/Apr/2017 00:37:43] DEBUG [paramiko.transport:282] [chan 1] Max packet in: 32768 bytes
[28/Apr/2017 00:37:43] DEBUG [paramiko.transport:282] [chan 0] EOF received (0)
[28/Apr/2017 00:37:43] DEBUG [paramiko.transport:282] [chan 0] EOF sent (0)
[28/Apr/2017 00:37:43] DEBUG [paramiko.transport:282] [chan 1] Max packet out: 32768 bytes
[28/Apr/2017 00:37:43] DEBUG [paramiko.transport:282] Secsh channel 1 opened.
[28/Apr/2017 00:37:43] DEBUG [paramiko.transport:282] [chan 1] Sesch channel 1 request ok
[28/Apr/2017 00:37:43] DEBUG [paramiko.transport:282] [chan 2] Max packet in: 32768 bytes
[28/Apr/2017 00:37:43] DEBUG [paramiko.transport:282] [chan 1] EOF received (1)
[28/Apr/2017 00:37:43] DEBUG [paramiko.transport:282] [chan 1] EOF sent (1)
[28/Apr/2017 00:37:43] DEBUG [paramiko.transport:282] [chan 2] Max packet out: 32768 bytes
[28/Apr/2017 00:37:43] DEBUG [paramiko.transport:282] Secsh channel 2 opened.
[28/Apr/2017 00:37:43] DEBUG [paramiko.transport:282] [chan 2] Sesch channel 2 request ok
[28/Apr/2017 00:37:43] DEBUG [paramiko.transport:282] [chan 2] EOF received (2)
[28/Apr/2017 00:37:43] DEBUG [paramiko.transport:282] [chan 2] EOF sent (2)
[28/Apr/2017 00:37:44] DEBUG [paramiko.transport:282] [chan 3] Max packet in: 32768 bytes
[28/Apr/2017 00:37:44] DEBUG [paramiko.transport:282] [chan 3] Max packet out: 32768 bytes
[28/Apr/2017 00:37:44] DEBUG [paramiko.transport:282] Secsh channel 3 opened.
[28/Apr/2017 00:37:44] DEBUG [paramiko.transport:282] [chan 3] Sesch channel 3 request ok
[28/Apr/2017 00:37:44] DEBUG [paramiko.transport:282] [chan 4] Max packet in: 32768 bytes
[28/Apr/2017 00:37:44] DEBUG [paramiko.transport:282] [chan 4] Max packet out: 32768 bytes
[28/Apr/2017 00:37:44] DEBUG [paramiko.transport:282] Secsh channel 4 opened.
[28/Apr/2017 00:37:44] DEBUG [paramiko.transport:282] [chan 4] Sesch channel 4 request ok
[28/Apr/2017 00:37:44] DEBUG [paramiko.transport:282] [chan 3] EOF sent (3)
[28/Apr/2017 00:37:44] DEBUG [paramiko.transport:282] [chan 4] EOF received (4)
[28/Apr/2017 00:37:44] DEBUG [paramiko.transport:282] [chan 4] EOF sent (4)
[28/Apr/2017 00:37:46] DEBUG [paramiko.transport:282] Ignoring message for dead channel 3
[28/Apr/2017 00:37:46] DEBUG [paramiko.transport:282] Ignoring message for dead channel 3
[28/Apr/2017 00:38:51] DEBUG [paramiko.transport:282] starting thread (client mode): 0x5a8e4250L
[28/Apr/2017 00:38:51] DEBUG [paramiko.transport:282] Local version/idstring: SSH-2.0-paramiko_2.1.2
[28/Apr/2017 00:38:51] DEBUG [paramiko.transport:282] Remote version/idstring: SSH-2.0-OpenSSH_6.6.1
[28/Apr/2017 00:38:51] INFO [paramiko.transport:282] Connected (version 2.0, client OpenSSH_6.6.1)
[28/Apr/2017 00:38:51] DEBUG [paramiko.transport:282] kex algos:[u'curve25519-sha256@libssh.org', u'ecdh-sha2-nistp256', u'ecdh-sha2-nistp384', u'ecdh-sha2-nistp521', u'diffie-hellman-group-exchange-sha256', u'diffie-hellman-group-exchange-sha1', u'diffie-hellman-group14-sha1', u'diffie-hellman-group1-sha1'] server key:[u'ssh-rsa', u'ecdsa-sha2-nistp256', u'ssh-ed25519'] client encrypt:[u'aes128-ctr', u'aes192-ctr', u'aes256-ctr', u'arcfour256', u'arcfour128', u'aes128-gcm@openssh.com', u'aes256-gcm@openssh.com', u'chacha20-poly1305@openssh.com', u'aes128-cbc', u'3des-cbc', u'blowfish-cbc', u'cast128-cbc', u'aes192-cbc', u'aes256-cbc', u'arcfour', u'rijndael-cbc@lysator.liu.se'] server encrypt:[u'aes128-ctr', u'aes192-ctr', u'aes256-ctr', u'arcfour256', u'arcfour128', u'aes128-gcm@openssh.com', u'aes256-gcm@openssh.com', u'chacha20-poly1305@openssh.com', u'aes128-cbc', u'3des-cbc', u'blowfish-cbc', u'cast128-cbc', u'aes192-cbc', u'aes256-cbc', u'arcfour', u'rijndael-cbc@lysator.liu.se'] client mac:[u'hmac-md5-etm@openssh.com', u'hmac-sha1-etm@openssh.com', u'umac-64-etm@openssh.com', u'umac-128-etm@openssh.com', u'hmac-sha2-256-etm@openssh.com', u'hmac-sha2-512-etm@openssh.com', u'hmac-ripemd160-etm@openssh.com', u'hmac-sha1-96-etm@openssh.com', u'hmac-md5-96-etm@openssh.com', u'hmac-md5', u'hmac-sha1', u'umac-64@openssh.com', u'umac-128@openssh.com', u'hmac-sha2-256', u'hmac-sha2-512', u'hmac-ripemd160', u'hmac-ripemd160@openssh.com', u'hmac-sha1-96', u'hmac-md5-96'] server mac:[u'hmac-md5-etm@openssh.com', u'hmac-sha1-etm@openssh.com', u'umac-64-etm@openssh.com', u'umac-128-etm@openssh.com', u'hmac-sha2-256-etm@openssh.com', u'hmac-sha2-512-etm@openssh.com', u'hmac-ripemd160-etm@openssh.com', u'hmac-sha1-96-etm@openssh.com', u'hmac-md5-96-etm@openssh.com', u'hmac-md5', u'hmac-sha1', u'umac-64@openssh.com', u'umac-128@openssh.com', u'hmac-sha2-256', u'hmac-sha2-512', u'hmac-ripemd160', u'hmac-ripemd160@openssh.com', u'hmac-sha1-96', u'hmac-md5-96'] client compress:[u'none', u'zlib@openssh.com'] server compress:[u'none', u'zlib@openssh.com'] client lang:[u''] server lang:[u''] kex follows?False
[28/Apr/2017 00:38:51] DEBUG [paramiko.transport:282] Kex agreed: diffie-hellman-group1-sha1
[28/Apr/2017 00:38:51] DEBUG [paramiko.transport:282] Cipher agreed: aes128-ctr
[28/Apr/2017 00:38:51] DEBUG [paramiko.transport:282] MAC agreed: hmac-sha2-256
[28/Apr/2017 00:38:51] DEBUG [paramiko.transport:282] Compression agreed: none
[28/Apr/2017 00:38:51] DEBUG [paramiko.transport:282] kex engine KexGroup1 specified hash_algo <built-in function openssl_sha1>
[28/Apr/2017 00:38:51] DEBUG [paramiko.transport:282] Switch to new keys ...
[28/Apr/2017 00:38:51] DEBUG [paramiko.transport:282] Adding ssh-rsa host key for 129.105.90.149: 3be932e37fd6c127bb6880a6ce0bab88
[28/Apr/2017 00:38:51] DEBUG [paramiko.transport:282] Trying discovered key 175016a8e15c3b3b3130d9f0396baec2 in /home/NANOMINE/.ssh/id_rsa
[28/Apr/2017 00:38:52] DEBUG [paramiko.transport:282] userauth is OK
[28/Apr/2017 00:38:52] INFO [paramiko.transport:282] Authentication (publickey) successful!
[28/Apr/2017 00:38:52] DEBUG [paramiko.transport:282] [chan 0] Max packet in: 32768 bytes
[28/Apr/2017 00:38:52] DEBUG [paramiko.transport:282] [chan 0] Max packet out: 32768 bytes
[28/Apr/2017 00:38:52] DEBUG [paramiko.transport:282] Secsh channel 0 opened.
[28/Apr/2017 00:38:52] DEBUG [paramiko.transport:282] [chan 0] Sesch channel 0 request ok
[28/Apr/2017 00:38:52] DEBUG [paramiko.transport:282] [chan 1] Max packet in: 32768 bytes
[28/Apr/2017 00:38:52] DEBUG [paramiko.transport:282] [chan 0] EOF received (0)
[28/Apr/2017 00:38:52] DEBUG [paramiko.transport:282] [chan 0] EOF sent (0)
[28/Apr/2017 00:38:52] DEBUG [paramiko.transport:282] [chan 1] Max packet out: 32768 bytes
[28/Apr/2017 00:38:52] DEBUG [paramiko.transport:282] Secsh channel 1 opened.
[28/Apr/2017 00:38:52] DEBUG [paramiko.transport:282] [chan 1] Sesch channel 1 request ok
[28/Apr/2017 00:38:52] DEBUG [paramiko.transport:282] [chan 2] Max packet in: 32768 bytes
[28/Apr/2017 00:38:52] DEBUG [paramiko.transport:282] [chan 1] EOF received (1)
[28/Apr/2017 00:38:52] DEBUG [paramiko.transport:282] [chan 1] EOF sent (1)
[28/Apr/2017 00:38:52] DEBUG [paramiko.transport:282] [chan 2] Max packet out: 32768 bytes
[28/Apr/2017 00:38:52] DEBUG [paramiko.transport:282] Secsh channel 2 opened.
[28/Apr/2017 00:38:52] DEBUG [paramiko.transport:282] [chan 2] Sesch channel 2 request ok
[28/Apr/2017 00:38:52] DEBUG [paramiko.transport:282] [chan 2] EOF received (2)
[28/Apr/2017 00:38:52] DEBUG [paramiko.transport:282] [chan 2] EOF sent (2)
[28/Apr/2017 00:38:53] DEBUG [paramiko.transport:282] [chan 3] Max packet in: 32768 bytes
[28/Apr/2017 00:38:53] DEBUG [paramiko.transport:282] [chan 3] Max packet out: 32768 bytes
[28/Apr/2017 00:38:53] DEBUG [paramiko.transport:282] Secsh channel 3 opened.
[28/Apr/2017 00:38:53] DEBUG [paramiko.transport:282] [chan 3] Sesch channel 3 request ok
[28/Apr/2017 00:38:53] DEBUG [paramiko.transport:282] [chan 4] Max packet in: 32768 bytes
[28/Apr/2017 00:38:53] DEBUG [paramiko.transport:282] [chan 4] Max packet out: 32768 bytes
[28/Apr/2017 00:38:53] DEBUG [paramiko.transport:282] Secsh channel 4 opened.
[28/Apr/2017 00:38:53] DEBUG [paramiko.transport:282] [chan 4] Sesch channel 4 request ok
